subsys/net/conn_mgr/conn_mgr_monitor.c

```diff
@@ -338,6 +338,9 @@ static int conn_mgr_mon_init(void)
 			CONFIG_NET_CONNECTION_MANAGER_MONITOR_STACK_SIZE,
 			conn_mgr_mon_thread_fn,
 			NULL, NULL, NULL, THREAD_PRIORITY, 0, K_NO_WAIT);
+	if (IS_ENABLED(CONFIG_THREAD_NAME)) {
```

Comment:
Right. I've seen both ways within the source. As this might not end up in production code, I could remove it.

Should I send in a new patch without this "if()" line?